### ISSUE TITLE
Introduce tools.keystrokeToArray method

### ISSUE THREAD

**Issue by mlewand** [ckeditor/ckeditor-dev#856]
## Are you reporting a feature request or a bug?

New feature

 ## Provide detailed reproduction steps (if any)

Currently [`keystrokeToString`](https://docs.ckeditor.com/#!/api/CKEDITOR.tools-method-keystrokeToString) method returns the whole keystroke as a string, like so:

```js
{
	display: '⌘ + X',
	aria: 'COMMAND + X'
}
```

For a better reusability it should return keys as an array, without joining it to a string with a "+" or any other character.

The easiest solution to that is to add `CKEDITOR.tools.keystrokeToArray` method, that would return an object like the following:

```js
{
	display: [ '⌘', 'X' ],
	aria: [ 'COMMAND', 'X' ]
}
```

Then `keystrokeToString` should reuse it (by simply joining array members using `+`).

As a follow-up ticket I'd like to improve list of keys supported by `keystrokeToString`/`keystrokeToArray` methods, as currently it supports [very limited amount of keys](https://github.com/ckeditor/ckeditor-dev/blob/da389f9f08a8df4562ce14ae02ad215dd510c454/lang/en.js#L101-L112), compared [to a11yhelp dialog](https://github.com/ckeditor/ckeditor-dev/blob/da389f9f08a8df4562ce14ae02ad215dd510c454/plugins/a11yhelp/dialogs/lang/en.js#L110-L163).

**Comment by Comandeer** [comment]
As we already have `keystrokeToString`, maybe it would be better to name the new function `keystrokeToArray` – just to be coherent?

**Comment by mlewand** [comment]
Seems like a good idea, I'll update the original report.

**Comment by mlewand** [comment]
Closed with #858.
